.github/dependabot.yml

```diff
@@ -0,0 +1,35 @@
+# *******************************************************************************
+# Copyright (c) 2024 Contributors to the Eclipse Foundation
+#
+#   See the NOTICE file(s) distributed with this work for additional
+#   information regarding copyright ownership.
+#
+#   This program and the accompanying materials are made available under the
+#   terms of the Apache License, Version 2.0 which is available at
+#   https://www.apache.org/licenses/LICENSE-2.0.
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#   WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#   License for the specific language governing permissions and limitations
+#   under the License.
+#
+#   SPDX-License-Identifier: Apache-2.0
+#   ********************************************************************************
```

Comment:
Please remove the extra whitespace after the `#` at the beginning of the lines for optical reasons.

Reply:
Fixed with : https://github.com/eclipse-tractusx/demand-capacity-mgmt/pull/82/commits/cd987593c66ce58a8b05440ac3ba3e2c5cc97fab